### ISSUE TITLE
Merit?

### ISSUE THREAD

**Issue by JohnyDL** [deltapi/abc-feedback#18]
ABC@home calculated merit too

> The merit is defined to be (q-1)^2(log r) log log r.

can we include this metric as it would seem that q tends to 1 and for people 'high score hunting' having a metric that has a chance of being bigger over time would give people more of a reason to stick around, there's a good chance I'll never see anything higher than my {"a":"184207739625","b":"976083480541134848","q":"1.3883856734866593"} triple as far as q goes but merit? For all I know one of my lower quality triples has higher merit

**Comment by Dinoguy1000** [comment]
The merit definition is from https://pub.math.leidenuniv.nl/~smitbde/abc/?set=0 (I tracked it down because the `2` in the formula looked suspicious to me and I wanted to confirm it was squaring `(q-1)`, not just doubling stuff).

Also, `r` is "the product of the distinct prime divisors of ABC" (from the same page); I don't think it's currently directly calculated (but could be wrong) or exposed to the player.

**Comment by JohnyDL** [comment]
You're totally right I did the copy paste but not check the square and that's the page I found too ^_^

But Log(r) is calculated (if not exposed to users) because that's part of the q formula and we're explicitly looking for abcs with q>1

**Comment by deltapi** [comment]
Thanks for thsuggestion, it is on the roadmap!
